mesh/mesh.go

```diff
@@ -142,6 +171,10 @@ func (m *Mesh) ValidateLayer(lyr *types.Layer) {
 	oldPbase, newPbase := m.HandleIncomingLayer(lyr)
 	m.lvMutex.Lock()
 	m.validatedLayer = currLayerId
+	m.validatedLayer = lyr.Index()
+	if err := m.general.Put(VALIDATED, lyr.Index().ToBytes()); err != nil {
+		m.Error("could not persist validated layer index")
```

Comment:
what should happen if we cant persist the layer? also add layerid to error...

Reply:
for now this behavior is enough, changed the log